uint8 armBasePos
uint8 armAPos
uint8 armBPos
uint8 gripperBasePos
uint8 gripperPos